staging/src/k8s.io/apiserver/pkg/storage/tests/cacher_test.go

```diff
@@ -466,12 +477,32 @@ func TestWatch(t *testing.T) {
 	_ = updatePod(t, etcdStorage, podFooBis, fooUpdated)
 
 	verifyWatchEvent(t, nowWatcher, watch.Modified, podFooBis)
+
+	// Make current watch cache full, 100 is watch cache initial capacity.
+	// Make start and last event duration exceed eventFreshDuration(current 75s) to ensure watch cache won't expand.
+	for i := 0; i < 100; i++ {
```

Comment:
use the constant instead of explicitly writing "100" (once we decrease it it will be updated for free)

Reply:
Do you mean a global constant in this test file?

Reply:
I mean `defaultLowerBoundCapacity`

Reply:
As they belongs to different packages, should a make a duplicated defaultLowerBoundCapacity here or make cacher.defaultLowerBoundCapacity public?

Reply:
aah - I misread it.

Let's make a constant in this file then and set to 100 with a comment that it should be kept in sync with defaultLowerBoundCapacity.